debian 10 编译报错

Please answer these questions before submitting your issue. Thanks!

(Please mention that if the issue you filed is solved, you may wish to close it by yourself. Thanks again.)

(PS, you can remove 3 lines above, including this one, before post your issue.)

### What version of shadowsocks-libev are you using?
3.3.4-1 cc658e4

### What operating system are you using?
Debian 10.3

### What did you do?
./autogen.sh && dpkg-buildpackage -b -us -uc

### What did you expect to see?


### What did you see instead?
`make[4]: Leaving directory '/root/ss-dev/shadowsocks-libev'
make[3]: Leaving directory '/root/ss-dev/shadowsocks-libev'
make[2]: Leaving directory '/root/ss-dev/shadowsocks-libev'
make[1]: Leaving directory '/root/ss-dev/shadowsocks-libev'
   dh_install
   dh_installdocs
   debian/rules override_dh_installchangelogs
make[1]: Entering directory '/root/ss-dev/shadowsocks-libev'
dh_installchangelogs -XChanges
make[1]: Leaving directory '/root/ss-dev/shadowsocks-libev'
   dh_installman
   dh_systemd_enable
   dh_installinit
   dh_systemd_start
   dh_lintian
   dh_perl
   dh_link
   dh_strip_nondeterminism
   dh_compress
   dh_fixperms
   debian/rules override_dh_missing
make[1]: Entering directory '/root/ss-dev/shadowsocks-libev'
dh_missing --fail-missing
dh_missing: usr/lib/x86_64-linux-gnu/libshadowsocks-libev.la exists in debian/tmp but is not installed to anywhere
	The following debhelper tools have reported what they installed (with files per package)
	 * dh_install: libshadowsocks-libev-dev (3), libshadowsocks-libev2 (2), shadowsocks-libev (18)
	 * dh_installdocs: libshadowsocks-libev-dev (0), libshadowsocks-libev2 (0), shadowsocks-libev (11)
	 * dh_installman: libshadowsocks-libev-dev (0), libshadowsocks-libev2 (0), shadowsocks-libev (0)
	If the missing files are installed by another tool, please file a bug against it.
	When filing the report, if the tool is not part of debhelper itself, please reference the
	"Logging helpers and dh_missing" section from the "PROGRAMMING" guide for debhelper (10.6.3+).
	  (in the debhelper package: /usr/share/doc/debhelper/PROGRAMMING.gz)
	Be sure to test with dpkg-buildpackage -A/-B as the results may vary when only a subset is built
	For a short-term work-around: Add the files to debian/not-installed
dh_missing: missing files, aborting
make[1]: *** [debian/rules:24: override_dh_missing] Error 2
make[1]: Leaving directory '/root/ss-dev/shadowsocks-libev'
make: *** [debian/rules:27: binary] Error 2
dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
`

### What is your config in detail (with all sensitive info masked)?

--- a/debian/rules
+++ b/debian/rules
@@ -20,8 +20,5 @@ override_dh_auto_configure:
 override_dh_installchangelogs:
 	dh_installchangelogs -XChanges
 
-override_dh_missing:
-	dh_missing --fail-missing
-
 %:
 	dh $@

